### DISCUSSION TITLE
窓設定レシピ集

### DISCUSSION THREAD

**Discussion by otiai10** [otiai10/demado#108] [Ideas]
# demadoのユーザがどんどん設定事例をアップしてくれる場所

「demadoの設定」画面のエクスポート機能を使って作成したjsonファイルを添付してくれるとうれしいです

**Comment by otiai10** [comment]
# 花騎士 x1.0
[export-1725103623105.json](https://github.com/user-attachments/files/16824514/export-1725103623105.json)

**Reply by otiai10** [reply]
花騎士、設定URL https じゃないとうまくいかないです。httpじゃなくて

**Comment by otiai10** [comment]
# 艦これ x0.5

[export-1724109279224.json](https://github.com/user-attachments/files/16666812/export-1724109279224.json)

**Comment by otiai10** [comment]
# ミストレx0.5

[export-1724335030006.json](https://github.com/user-attachments/files/16711676/export-1724335030006.json)

**Comment by OinuGames** [comment]
## demado 2.0.60版 DMM/FANZAのゲーム設定 [ズーム: 1.0]
#### 動作確認: Windows11 25H2 + Google Chrome
・一部のゲームはスクロールバー等を消せず、残したままサイズ調整しています
|ゲーム タイトル|DMM JSON|FANZA JSON|
|:----:|:----:|:----:|
|艦隊これくしょん|[20251019_kancolle](https://github.com/user-attachments/files/28331641/20251019_kancolle.json)| - |
|御城プロジェクト:RE|[20250222_oshirore](https://github.com/user-attachments/files/28331646/20250222_oshirore.json)| - |
||||
|**ゲーム タイトル**|**DMM JSON**|**FANZA JSON**|
|千年戦争アイギス|[20250131_aigisc](https://github.com/user-attachments/files/28331667/20250131_aigisc.json)|[20250222_aigis](https://github.com/user-attachments/files/28331677/20250222_aigis.json)|
|FLOWER KNIGHT GIRL|[20250304_flower](https://github.com/user-attachments/files/28331687/20250304_flower.json)|[20250304_flower-x](https://github.com/user-attachments/files/28331691/20250304_flower-x.json)|
|神姫PROJECT|[20250321_kamipro](https://github.com/user-attachments/files/28331699/20250321_kamipro.json)|[20250321_kamipror](https://github.com/user-attachments/files/28331703/20250321_kamipror.json)|
|あやかしランブル！|[20250321_ayarabu](https://github.com/user-attachments/files/28331708/20250321_ayarabu.json)|[20250321_ayarabux](https://github.com/user-attachments/files/28331711/20250321_ayarabux.json)|
|天啓パラドクス|[20250131_tenkeiprdx](https://github.com/user-attachments/files/28331719/20250131_tenkeiprdx.json)|[20250131_tenkeiprdx_x](https://github.com/user-attachments/files/28331724/20250131_tenkeiprdx_x.json)|
|モンスター娘TD|[20250304_monmusutd](https://github.com/user-attachments/files/28331733/20250304_monmusutd.json)|[20250304_monmusutdx](https://github.com/user-attachments/files/28331738/20250304_monmusutdx.json)|
|ガールズクリエイション|[20250321_girlscreation](https://github.com/user-attachments/files/28331746/20250321_girlscreation.json)|[20250321_girlscreationr](https://github.com/user-attachments/files/28331756/20250321_girlscreationr.json)|
|マブラヴ ガールズガーデン|[20250916_muvluv_gg](https://github.com/user-attachments/files/28331777/20250916_muvluv_gg.json)|[20250916_muvluv_ggx](https://github.com/user-attachments/files/28331779/20250916_muvluv_ggx.json)|
|ドットアビス|[20260414_dotabyss](https://github.com/user-attachments/files/28331794/20260414_dotabyss.json)|[20260414_dotabyss_x](https://github.com/user-attachments/files/28331795/20260414_dotabyss_x.json)|
||||
|**DMM 27件一括<BR>FANZA 27件一括**|[20260528__DMM](https://github.com/user-attachments/files/28331861/20260528__DMM.json)|[20260528__FANZA](https://github.com/user-attachments/files/28331863/20260528__FANZA.json)|

**Comment by SuwakoFrog** [comment]
# FANZA版 x1.0
| ゲーム名 | レシピ |
|:----------:|:----------:|
| FLOWER KNIGHT GIRL X | [fkgX.json](https://github.com/user-attachments/files/16931078/fkgX.json) |
| あやかしランブル X | [ayarabuX.json](https://github.com/user-attachments/files/16931082/ayarabuX.json) |
| エンジェリックリンク R | [angeliclinkR.json](https://github.com/user-attachments/files/16931086/angeliclinkR.json) |
| ミストトレインガールズ X | [misttrainX.json](https://github.com/user-attachments/files/16931087/misttrainX.json) |
| モンスター娘TD X | [monmusutdX.json](https://github.com/user-attachments/files/16931090/monmusutdX.json) |
| 戦国†恋姫オンライン X| [senkoiX.json](https://github.com/user-attachments/files/16931096/senkoiX.json) |
| ルミナプログノシス X | [lumiproX.json](https://github.com/user-attachments/files/16931101/lumiproX.json) |

**Comment by SHOU-AZUMA** [comment]
# **FANZAｹﾞｰ**

| DeepOne 虚無と夢幻のフラグメント R  | [tonofuraR.json](https://github.com/user-attachments/files/17381450/tonofuraR.json) |
| ------------- | ------------- |
| りりぃあんじぇ  | [lilyange.json](https://github.com/user-attachments/files/17382508/lilyange.json)  |
| エデンズリッターグレンツェ X | [denzuri.json](https://github.com/user-attachments/files/17382567/denzuri.json) |

**Reply by SHOU-AZUMA** [reply]
とのフラは x1.1です

**Comment by prinny0142** [comment]
### 対魔忍RPG　FANZA
 [taimanin.json](https://github.com/user-attachments/files/17822035/taimanin.json)
アドレスバー非表示だとうまく表示されません。

**Comment by vincere491** [comment]
### 艦これ（2025/10セキュア化以降）

[export-1760756470598.json](https://github.com/user-attachments/files/22981906/export-1760756470598.json)
艦これウィジェットが対応しなくなったため、暫定的な対応としてどうぞ

**Comment by yajiusa-creator** [comment]
### **刀剣乱舞** x1.0　DMM
[export-1780010232364.json](https://github.com/user-attachments/files/28370977/export-1780010232364.json)
### **文豪とアルケミスト**x0.9　DMM
[export-1780010666739.json](https://github.com/user-attachments/files/28370990/export-1780010666739.json)

自分用で恐縮ですが置かせてください
